### PR TITLE
22802: GlueDB PR task related to MA file naming work

### PR DESCRIPTION
**Pull request by prem548**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [22802](https://devops.dchbx.org/redmine/issues/22802) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | @ap-dc |

### CODE REVIEW

**Review by bill-transue — APPROVED** [review]
This looks correct, should be able to find the correct file names now.

File: script/queries/steven_new.rb [hunk 1]
```diff
@@ -15,13 +17,15 @@ def bad_eg_id(eg_id)
...
-
-environment_name = hostname.gsub(".","").gsub("edidbmhchbxshoporg","")
-
 timestamp = Time.now.strftime('%Y_%m_%d_%H_%M_%S')
 
-filename = "CCA_#{environment_name}_enrollment_#{timestamp}.csv"
+filename = if individual_market_is_enabled?
+             hostname = %x(echo $HOSTNAME).strip
+             environment_name = hostname.gsub(".","").gsub("edidbmhchbxshoporg","")
+             "CCA_#{environment_name}_enrollment_#{timestamp}.csv"
```
Comment (Kevin-Wei) [inline script/queries/steven_new.rb]: This isn't the filename in DC

**Review by Kevin-Wei — CHANGES_REQUESTED** [review]

File: config/settings.yml [hunk 2]
```diff
@@ -15,3 +14,6 @@ contact_center:
   postal_code: "02114"
 
 abbrev: "mhc"
+
+aca:
+  enrollment_policy_report_name: "CCA_PRODUCTION_ENROLLMENT"
```
Comment (bill-transue) [inline config/settings.yml:19]: Nice, now it can be set to whatever.

**Review by bill-transue — APPROVED** [review]
Good job moving the report name to the `config/settings.yml` and nice spec for testing it.

**Pull request merged**